server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -473,10 +473,10 @@ cics_invoke: INVOKE (SERVICE cics_data_value | CHANNEL cics_data_value | OPERATI
 
 /** ISSUE (all of them) */
 cics_issue:
-	ISSUE (cics_issue_print | cics_issue_eods | cics_issue_abend | cics_issue_abort | cics_issue_add | cics_issue_confirmation
+	ISSUE (cics_issue_print | cics_issue_wait | cics_issue_eods | cics_issue_abend | cics_issue_abort | cics_issue_add | cics_issue_confirmation
```

Comment:
I tried using `ISSUE WAIT ERASEAUP` and it was rejected by the cics pre-processor, could you re-check if perhaps some of the other keywords are required to be in the first position?

Reply:
Yes, it is invalid, but `cics_issue_erase_aup` will accept it. `ISSUE ERASEAUP WAIT` is however valid.